src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -44,11 +44,12 @@ PreviewCard.propTypes = {
   closeModal: PropTypes.func.isRequired,
   addConcept: PropTypes.func.isRequired,
   concept: PropTypes.shape({
-    display_name: PropTypes.string.isRequired,
-    descriptions: PropTypes.array.isRequired,
```

Comment:
Arrays take a shape describing what it contains. Check out this [tread](https://stackoverflow.com/questions/32325912/react-proptype-array-with-shape)

Reply:
@dessHub , according to React 16.5.2, it is the correct way of doing it. [Check this out](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes)

Reply:
It should be `PropTypes.arrayOf()`

Reply:
@dessHub I think that you use `PropTypes.arrayOf()` if it is an array of objects.